test/sys/test_socket.rs

```diff
@@ -57,6 +57,64 @@ pub fn test_inetv4_addr_roundtrip_sockaddr_storage_to_addr() {
     assert_eq!(from_storage, sockaddr);
 }
 
+#[cfg(any(target_os = "linux"))]
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
```

Comment:
Why limit the target_arch ?

Reply:
I'm assuming you mean it fails in CI.  But on which platforms?  It's probably a QEMU thing, in which case you should use `#[cfg_attr(qemu, ignore)]` .

Reply:
Seems to work.